### DISCUSSION TITLE
Render from string?

### DISCUSSION THREAD

**Discussion by SamirKharchi** [cefsharp/CefSharp#3685] [General]
Hi,

I am trying to figure out if CefSharp.Wpf allows to render an HTML page from a string source (not an URL) but cannot find any info on that. Does anybody know how to accomplish that without writing the stuff to disk and use that as an URL?

Cheers
Sam

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#loading-htmlcssjavascriptetc-from-diskdatabaseembedded-resourcestream

**Reply by SamirKharchi** [reply]
Thanks! Exactly what I was looking for.
